ccpp/suites/suite_FV3_RRFS_v1alpha.xml

```diff
@@ -1,84 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
While updating the SPP PR to the latest fv3atm code, I noticed this file was deleted (along with a few other regional-based suites: FV3_GSD_v0, FV3_GSD_noah, and FV3_GSD_SAR). This particular suite is one of the two main SDFs in the SRW App release v1.0, and we should keep it for now.  FYI: @jwolff-ncar, @gsketefian, @mkavulich @BenjaminBlake-NOAA @chan-hoo.

Reply:
This was discussed with Curtis and others. The reasoning for removing it is that this suite is not going to be the operational RRFS suite in the future. Since code bases for the SRW App 1.0 and develop have diverged so far, it is not realistic to run the SRW App code with the head of the develop branches. If users have problems with the public release using RRFS v1alpha, you can't just point them to the develop branches and solve their questions there, because the code is vastly different. Therefore there is no harm in cleaning up old suite definition files.

It was decided to retain RRFS v1 beta for now. And many of the GSD_* suites have simply been renamed and rebased on the RAP suite.

Reply:
@climbfuji, thanks for providing some background information.  Unfortunately, the DTC team that works on SRW App code management, development and UFS-R2O RRFS testing wasn't aware of this discussion or decision.  

I'm certainly in favor of cleaning up old SDFs, but we just need to be careful.  In this case, since the v1beta SDF differs only in its use of MYNN SFC instead of GFS SFC, the move to v1beta makes sense and the authoritative code has diverged significantly from the release, as you mentioned.  However, we've done a lot of RRFS benchmarking UFS-R2O work in the last few months with the v1alpha SDF in the latest SRW App, so I figured it'd be good for users to still have it available in the latest authoritative code for testing/comparison to those results.  If that's not possible, then OK.  

I would be good to find a way to get all relevant voices included on important SRW App decisions like these in the future.

Reply:
Are you pointing to the latest develop branches in the app? If not (e.g. if you are using your own forks), then you can simply retain the file for testing/benchmarking. It's a challenge to know for the non-regional people who needs to be consulted for which kind of changes. In this case, Curtis, Christina and several others (that I don't remember) were involved.